contracts/pcv/sentinel/guards/examples/BalanceGuard.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "../../IGuard.sol";
+
+contract BalanceGuard is IGuard {
+    address constant private ZERO = address(0x0);
+
+    function check() 
+        external 
+        view
+        override 
+        returns (bool) 
+    {
+        if(address(this).balance % 2 == 0) return true;
```

Comment:
assuming this is just a test?

Reply:
yeah, it lets me send eth to it and change the trigger condition